platform/ios/CHANGELOG.md

```diff
@@ -8,6 +8,9 @@ Mapbox welcomes participation and contributions from everyone. Please read [CONT
 
 * Fixed flickering on style change for the same tile set ([#15127](https://github.com/mapbox/mapbox-gl-native/pull/15127))
 
+### Other changes
+
+* Improved feature querying performance (focussed on roads). ([#15183](https://github.com/mapbox/mapbox-gl-native/pull/15183))
```

Comment:
focused?

Reply:
UK English ;)

Reply:
If even the Europeans are against you... 😅

In any case, this also struck me as wrong. I think we should stick with ‘merican English, given our audience/company/etc.

Reply:
Alternatively:

```suggestion
* Improved feature querying performance of road features. ([#15183](https://github.com/mapbox/mapbox-gl-native/pull/15183))
```